orderer/common/types/channelinfo.go

```diff
@@ -29,6 +29,23 @@ type ChannelInfoShort struct {
 	URL string `json:"url"`
 }
 
+type ClusterRelation string
+
+const (
+	ClusterRelationMember        ClusterRelation = "member"
+	ClusterRelationFollower      ClusterRelation = "follower"
+	ClusterRelationConfigTracker ClusterRelation = "config-tracker"
```

Comment:
what's a config tracker? I don't see anything about it in the RFC

Reply:
This indeed was not in the RFC because the RFC did not address how this API would behave on a "legacy" system that runs in the system channel model. We have to add it to the RFC some time for completeness. 

The ClusterRelation="config-tracker" is the inactive.Chain. It shows up in the "List" of channels, and We have to differentiate it from the etcdraft.Chain which is "member"/"active". 

The ClusterRelation="none" is for kafka and solo. I know they are deprecated, but they are still in the code-base, so I addressed it.

It is all written on the documentation of the respective ClusterRelation() method, but I guess it needs to be on the constant as well.

Reply:
Makes sense, thanks